runtime/gc_glue_java/ConcurrentMarkingDelegate.cpp

```diff
@@ -171,22 +124,13 @@ MM_ConcurrentMarkingDelegate::scanThreadRoots(MM_EnvironmentBase *env)
 		}
 	}
 
-	markSchemeStackIteratorData localData;
-	localData.concurrentMarkingDelegate = this;
-	localData.env = env;
-	/* In a case this thread is a carrier thread, and a virtual thread is mounted, we will scan virtual thread's stack. */
-	GC_VMThreadStackSlotIterator::scanSlots(vmThread, vmThread, (void *)&localData, concurrentStackSlotIterator, true, false);
+	MM_MarkingDelegate::StackIteratorData localData(env, _markingDelegate);
+	const bool stackFrameClassWalkNeeded = true;
+	GC_VMThreadStackSlotIterator::scanSlots(vmThread, vmThread, (void *)&localData, (J9MODRON_OSLOTITERATOR *)_markingDelegate->stackSlotIterator, stackFrameClassWalkNeeded, false);
```

Comment:
Not sure if our coding standard prefer anything, but for static methods I prefer `MM_MarkingDelegate::stackSlotIterator` style